NuimoSDK/project.json

```diff
@@ -1,16 +0,0 @@
-﻿{
-  "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.0.0"
-  },
-  "frameworks": {
-    "uap10.0": {}
-  },
-  "runtimes": {
-    "win10-arm": {},
-    "win10-arm-aot": {},
-    "win10-x86": {},
-    "win10-x86-aot": {},
-    "win10-x64": {},
-    "win10-x64-aot": {}
-  }
-}
```

Comment:
No it's not necessary. It's a required file for the project type you initially created. It is not used by the project type I changed to.